### PR TITLE
Remove todo to consider adding the cronjob name as a label

### LINKED ISSUE #41633: CronJob should apply a label to each created job
**What keywords did you search in Kubernetes issues before filing this one?** (If you have found any duplicates, you should instead reply there.):

"scheduledjob" (and try to understand from the title whether this could be related to my issue)
"find job"

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Feature request.

**Kubernetes version** (use `kubectl version`):
~~~~
Client Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.2", GitCommit:"08e099554f3c31f6e6f07b448ab3ed78d0520507", GitTreeState:"clean", BuildDate:"2017-01-12T04:57:25Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.1+coreos.0", GitCommit:"cc65f5321f9230bf9a3fa171155c1213d6e3480e", GitTreeState:"clean", BuildDate:"2016-12-14T04:08:28Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
~~~~

**Environment**:
- **Cloud provider or hardware configuration**: AWS via kube-aws 0.93-ish

**What happened**:
I'm using scheduled jobs, and then try to review the logs of the job executions. I have multiple scheduled jobs in the system. Generally this works via:
1. `kubectl get job` ... search for the ones that seem to relate to my scheduled job name
2. `kubectl get pod -l job-name=(name of the job that looks most recent from the previous list)`
3. `kubectl logs (name of the pod)`. Repeat this if there are more than one execution.
4. Repeat the whole list for previous jobs

**What you expected to happen**:
I'd love to be able to filter the initial job list already to only include jobs created by a specific scheduled job, something like `kubectl get job -l scheduledjob=NAME`. For that to happen I need scheduledjob to label the created job, just like a job already labels the pods executing it.

### PR DESCRIPTION
**Pull request by dmathieu**
**What this PR does / why we need it**:

It seems this label shouldn't be added automatically. If so, we should remove the comment.
See #59473

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #41633

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/cronjob/utils.go [hunk 1]
```diff
@@ -173,7 +173,6 @@ func getRecentUnmetScheduleTimes(sj batchv1beta1.CronJob, now time.Time) ([]time
 func getJobFromTemplate(sj *batchv1beta1.CronJob, scheduledTime time.Time) (*batchv1.Job, error) {
 	// TODO: consider adding the following labels:
 	// nominal-start-time=$RFC_3339_DATE_OF_INTENDED_START -- for user convenience
```
Comment (dmathieu) [inline pkg/controller/cronjob/utils.go]: We may want to remove this one too

**Review by kow3ns — APPROVED** [review]

**Comment by thockin** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/59542)

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

File: pkg/controller/cronjob/utils.go (hunk 1 above)
Comment (janetkuo) [reply pkg/controller/cronjob/utils.go]: Suggest removing this whole TODO, since we won't be adding labels to jobs

Comment (janetkuo) [inline pkg/controller/cronjob/utils.go]: Suggest removing this whole TODO, since we won't be adding labels to jobs

Comment (dmathieu) [reply pkg/controller/cronjob/utils.go]: Yup, I suggested that in another comment as well, but didn't get anything back.

**Review by janetkuo — APPROVED** [review]

**Pull request merged**